dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -240,8 +240,8 @@ public String buildInsertCommand(String tableName,
         return sb.toString();
     }
 
-    public DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors) {
-        return new DbStoredProcedureCall(this, name, accessors);
+    public DbStoredProcedure newDbStoredProcedure(String name) {
```

Comment:
So we're passing the accessors at later stage... I'm not sure we want that. We're not supporting that yet but it's possible to have multiple routines with same name and differing in their parameters or return value only.
Besides, in `buildInsertPreparedStatement` we're using the approach with passing the accessors.